GOVERNANCE.md

```diff
@@ -206,11 +229,22 @@ push commits, etc.), so what's the minimal amount is subjective, and there will
 be cases where collaborators disagree on whether a nomination should move
 forward.
 
+An important rule of thumb is that the nomination process is intended to be
+biased strongly towards implicit approval of the nomination. This means
+discussion and review around the proposal should ideally be more geared
+towards "I have reasons to say no..." as opposed to "Give me reasons to say
+yes...".
+
 When concerns have been raised on the private discussion, refrain from opening
 the public issue. If no one has explicitly blocked the nomination and you'd like
 it to move forward, comment something like "If I don't hear any objections
 before (some time), I will open the public issue".
 
+Also refrain from discussing or debating aspects of the nomination process
+itself directly within a nomination private discussion or public issue.
+Such discussions can derail and frustrate the nomination and cause unnecessary
+friction. Move such discussions to a separate issue or discussion thread.
```

Comment:
Just a heads-up  that this is being moved to the previous section in https://github.com/nodejs/node/pull/57483